app/resources/v1/street_images.js

```diff
@@ -3,11 +3,21 @@ const config = require('config')
 const User = require('../../models/user.js')
 const Street = require('../../models/street.js')
 const logger = require('../../../lib/logger.js')()
+const { SAVE_THUMBNAIL_EVENTS } = require('../../../lib/util.js')
 
 const ALLOW_ANON_STREET_THUMBNAILS = false
 
 exports.post = async function (req, res) {
-  const image = req.body
+  let json
+
+  try {
+    json = await JSON.parse(req.body)
+  } catch (error) {
+    res.status(400).send('Could not parse body as JSON.')
```

Comment:
Can you send this error as a JSON object instead of a string?

Reply:
That's what I attempted to do at first, but remember how we changed the POST request `/api/v1/street/images` to accept `text/plain` because the data URL was too large and for some reason I could only increase the limit size for one endpoint using `bodyParser.text({ limit })`? 

I tried again to send the JSON object as a `application/json` but then we end up with the `request entity too large` error. If I try to limit only the POST `/api/v1/street/images` endpoint by doing `bodyParser.json({ limit })` it doesn't work at all.

Reply:
That only affects the request payload, right? Does that mean you can't respond with a JSON object as the `400` status code?

Reply:
OH sorry I misunderstood the comment. Yes, I can do that. Sorry!

Reply:
Yes - although it might be a good idea to add that as a comment in the code so that in the future we remember why it needs to be sent as a stringified JSON and not a JSON directly.